src/storage/redis_db.cc

```diff
@@ -708,7 +708,7 @@ rocksdb::Status Database::typeInternal(const Slice &key, RedisType *type) {
   return rocksdb::Status::OK();
 }
 
-rocksdb::Status Database::Move(const std::string &key, const std::string &new_key, bool nx, bool *ret,
+rocksdb::Status Database::Move(const std::string &key, const std::string &new_key, bool nx, bool delete_old, bool *ret,
                                bool *key_exist) {
```

Comment:
Please unify `ret` and `key_exist` to just one output value.

Reply:
How to do it, use int?

Reply:
```
enum { KEY_NOT_EXIST, KEY_ALREADY_EXIST, DONE }
```